views/jira-manual-app-creation.hbs

```diff
@@ -163,13 +163,15 @@
         <input type="hidden" id="gitHubBaseUrl" name="gitHubBaseUrl" value="{{serverUrl}}">
 
         <div class="jiraManualAppCreation__serverError">
-          {{> server-error-message-box
-              title="We couldn't create your GitHub app"
-              message="Please make sure all the details you entered are correct."
-          }}
+          {{> server-error-message-box }}
         </div>
 
-        <input type="submit" value="{{#if app.id}}Update{{else}}Next{{/if}}" class="jiraManualAppCreation_formSubmitBtn aui-button aui-button-primary" />
+        <input
```

Comment:
One thing is remaining, when the user edits an app, then the form should display a file name instead of the blank _Drag and Drop_ area. .
he value for the input file `privateKeyFile` needs to set, otherwise the update button won't work as the form is not valid.

Reply:
Gotcha. I'd just been uploading the file again. Will update.

Reply:
So when updating, are we expecting user to upload the `privateKey` file again?
Depends on that, we may or may not want to "set the value of the input file", and also affect how we handle it in the db saving model, does it allow partial save or always full save?

Reply:
@krazziekay seeing we don't have the name of the file, were you thinking we should instead use the name of the app? e.g. my-app.pem?

Reply:
We can use any file name as discussed [here](https://atlassian.slack.com/archives/C039YPCGCSG/p1658795822188309). 
The only thing that the user can do is just see. If the user wants to change it, then they'll have to remove the current one and add the new one.
So, just need to check if there is existing private key, and if there is, we simply show a file already added instead of the blank file input, with any name. 
And we would also need to pass the saved private key as it is if the user hasn't upload any new file, when hitting submit.

Reply:
Checking with design to see what their thoughts are from a UX standpoint.